src/index.js

```diff
@@ -18,21 +18,45 @@ const TCP = and(IP, base('tcp'))
 const UDP = and(IP, base('udp'))
 const UTP = and(UDP, base('utp'))
 
+const TCP_IPFS = or(
```

Comment:
What is the use case for `TCP_IPFS`, utility method? This might replicate faster, why not do

```
mafmt.IPFS(ma) && mafmt.TCP(ma)
```